backend/src/main/java/com/github/makitango/curiocity/event/services/EventService.java

```diff
@@ -29,10 +34,12 @@ public Event addEvent(EventDTO eventDTO) {
         return eventRepository.save(entityEvent);
     }
 
-    public Event getEventById(String id) {
-        return eventRepository.findById(id).orElseThrow(() -> new RuntimeException("Event not found"));
+    public Event getEventById(String eventId) {
+        Optional<Event> eventOptional = eventRepository.findById(eventId);
+        return eventOptional.orElseThrow(EventNotFoundException::new);
```

Comment:
Eine Zeile reicht hier
```suggestion
    public Event getEventById(String eventId) {
        return eventRepository.findById(eventId).orElseThrow(EventNotFoundException::new);
```